<a href="https://colab.research.google.com/github/ariel-ortiz/202013-tc1028.401/blob/master/1.%20Fundamentos/Cadenas_con_Formato.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Cadenas con formato

In [1]:
def email(nombre: str, dominio: str) -> str:
    resultado = f'{nombre}@{dominio}'
    return resultado

In [2]:
email('pepito', 'example.com')

'pepito@example.com'

In [3]:
def nombre_completo(nombre: str, apellido_paterno: str, apellido_materno) -> str:
    resultado = f'{apellido_paterno} {apellido_materno}, {nombre}'
    return resultado

In [4]:
nombre_completo('Juan', 'Camaney', 'Pérez')

'Camaney Pérez, Juan'

In [5]:
def sufijo_longitud(frase: str) -> str:
    resultado = f'{frase}{len(frase)}'
    return resultado

In [6]:
sufijo_longitud('unElafante')

'unElafante10'

In [7]:
f'{2*3:10}'

'         6'

In [8]:
x = 'Hola'
f'{x:^10}'

'   Hola   '

In [9]:
titulo = 'Tec de Monterrey'
nombre = 'Ariel Ortiz'
print(f'{titulo:*^50}')
print(f'{nombre:~^50}')

*****************Tec de Monterrey*****************
~~~~~~~~~~~~~~~~~~~Ariel Ortiz~~~~~~~~~~~~~~~~~~~~


In [10]:
x = 12341234
print(f'{x:,}')

12,341,234


In [11]:
money = 12345.1294
print(f'{money:,.2f}')
print(f'{money:.5e}')

12,345.13
1.23451e+04


El número anterior en formato $\LaTeX$ se escribe así:
$$
1.2345 \times 10 ^ {4}
$$

In [12]:
n = 5
print(f'{n:0>10}')

0000000005
